hbase-server/src/main/java/org/apache/hadoop/hbase/master/ServerManager.java

```diff
@@ -921,8 +921,10 @@ public boolean isClusterShutdown() {
   public void startChore() {
     Configuration c = master.getConfiguration();
     if (persistFlushedSequenceId) {
-      // when reach here, RegionStates should loaded, firstly, we call remove deleted regions
-      removeDeletedRegionFromLoadedFlushedSequenceIds();
+      new Thread(() -> {
+        // when reach here, RegionStates should loaded, firstly, we call remove deleted regions
+        removeDeletedRegionFromLoadedFlushedSequenceIds();
+      }, "RemoveDeletedRegionSyncThread").start();
```

Comment:
Should this be a demon thread?
And seems that removeDeletedRegionFromLoadedFlushedSequenceIds could be a part of FlushedSequenceIdFlusher?

Reply:
> Should this be a demon thread?
no need, hmaster won't need the thread when it's done...
> And seems that removeDeletedRegionFromLoadedFlushedSequenceIds could be a part of FlushedSequenceIdFlusher?
no, removeDeletedRegionFromLoadedFlushedSequenceIds takes a long time and is disposable. if in FlushedSequenceIdFlusher, FlushedSequenceIdFlusher takes up choreService for a long time...Other tasks of choreService 
 cannot be executed.

Reply:
The complexity of removeDeletedRegionFromLoadedFlushedSequenceIds is O(n) when there are n regions. Why it takes a long time?

Reply:
n is 50W,100W...so removeDeletedRegionFromLoadedFlushedSequenceIds is very slow.